src/ralph/backends/storage/fs.py

```diff
@@ -0,0 +1,130 @@
+"""FileSystem storage backend for Ralph"""
+
+import datetime
+import logging
+import os
+import sys
+
+from ..mixins import HistoryMixin
+from .base import BaseStorage
+
+logger = logging.getLogger(__name__)
+
+
+class FSStorage(HistoryMixin, BaseStorage):
+    """FileSystem storage backend"""
+
+    name = "fs"
+
+    _fspath = "fs_archives/"
+
+    def __init__(self):
+        if not os.path.isdir(self._fspath) and not os.path.exists(self._fspath):
+            logger.debug(
+                "No existing FileSystemStorage, creating a new one at %s", self._fspath
+            )
+            os.mkdir(self._fspath)
```

Comment:
You should use a recursive path creation instead in case intermediary folders do not exist.

Reply:
Done!